cytosnake/guards/ext_guards.py

```diff
@@ -0,0 +1,50 @@
+"""
+module: ext_guards.py
```

Comment:
I'm a bit confused by the strategy/philosophy of this file. Why does `CytoSnake` need to guard against this at all? Isn't our philosophy to have the tools that `CytoSnake` uses handle this? (e.g. pycytominer cytomining/pycytominer#262)

Reply:
It's true that `pycytominer` has its internal checking systems for file extensions. However, this module is mainly used for `CytoSnake` pathing system and potentially other software packages that may not check for format extensions. 

Could be that this module will removed in the future.

Also, I may have written this module before (pycytominer cytomining/pycytominer#262) I do not remember.

Reply:
Sounds good! Thanks for thinking this through! Will be good to keep in mind for future